src/perception/long_range_cam/long_range_cam.cpp

```diff
@@ -0,0 +1,86 @@
+#include "long_range_cam.hpp"
+#include <opencv2/imgproc.hpp>
```

Comment:
All include should go into the "pch.hpp" file

If you are curious, this is called a pre-compiled header and it speeds up successive compilations.